packages/apollo-server/src/express.ts

```diff
@@ -20,16 +21,17 @@ export class ApolloServer extends ApolloServerBase<
     const { app, request } = config;
 
     app.use(config.path, cors(config.cors), json(), (req, res, next) => {
+      // only show graphiql if we want html for a GET
+      const accept = accepts(req);
+      const types = accept.types() as string[];
+      const isHTML =
```

Comment:
Can we change the name of this variable to `prefersHtml` to better explain what the `find` is doing?

Reply:
I like it! Will do